### ISSUE TITLE
Adjust priorities of chemical identifiers

### ISSUE THREAD

**Issue by nlharris** [biolink/biolink-model#425]
**Is your feature request related to a problem? Please describe.**
Need to align priority order to ID prefixes with Vlado's prioritization: https://docs.google.com/spreadsheets/d/1dhuoIQzqZ1Hpph5q0ER_kmBLRplsleN3fLQGEi_yR_8/edit#gid=237874243

**What working group (or team) did this request originate from?**
Data Modeling

(Note: related to #424)

**Comment by deepakunni3** [comment]
Currently we have the following as priority order:
```
CHEBI
CHEMBL.COMPOUND
DRUGBANK
PUBCHEM.COMPOUND
MESH
HMDB
INCHI
INCHIKEY
UNII
KEGG
gtpo
```

Once the discussion on aligning priority is complete, we can rearrange the above list accordingly.

**Comment by nlharris** [comment]
I just noticed that this is redundant with #340. Should we close one of them?

**Comment by vdancik** [comment]
Proposed chemical-identifier prefixes and their order:

PubChemCID: (was PUBCHEM.COMPOUND)
ChEMBL: (was CHEMBL.COMPOUND)
UNII:
ChEBI:
DrugBank:
MeSH:
CAS:
ChemBank:
DrugCentral:
GtoPdb:
HMDB:
KEGG:
AEOLUS:
PharmGKB:
PubChemSID:
SIDER:
InChI:
InChIKey:
SMILES:

**Comment by newgene** [comment]
@deepakunni3 @vdancik any reason why InchiKey is considered a lower priority in both your lists? Isn't InchiKey created for the purpose of the chemical identifier (as a db-neutral identifier for chemical)? I could be missing some important facts here.

**Comment by deepakunni3** [comment]
Yes, InChiKey has been a reliable unique identifier in my experience as well. Curious to hear from @vdancik on the pros and cons. Is the priority low because not all knowledge sources provide InChiKey?

**Comment by vdancik** [comment]
The rationale for InChI, InChIKey, and SMILES being at the bottom of the list is that they are considered compound structure representations rather then identifiers. The drawback of InChiKey, being a hash for InChI, is that it may (theoretically) be not unique.

**Comment by newgene** [comment]
But isn't structure-based "identifiers" are the ultimate non-ambiguous representation of a chemical, especially if we want to map chemicals across databases? Choosing something structure-based and database-neutral is even preferred in my opinion. For example, what if a chemical from a paper is not included in any database yet, but its structure is most likely available.

**Comment by micheldumontier** [comment]
InChI is certainly a reliable identifier, and to one that is likely to have fewer problems than mappings created for database-centric identiers. However, InChI won't work for chemical substances of undefined molecular composition. You'll have to rely on another database (or ontology) identifier for that.

**Comment by vdancik** [comment]
Structures to compounds are like sequences to genes - while they characterize compounds/genes, they might not be the most suitable identifiers. There is a reason why DrugBank, PubChem, ChEMBL, ChEBI, etc. have introduced their identifiers and do not use structure representation as an id.

**Comment by nlharris** [comment]
Vlado says this was addressed in Biolink 2.0 and can be closed.
